src/hera/workflow.py

```diff
@@ -320,3 +338,20 @@ def get_parameter(self, name: str) -> Parameter:
         if self.parameters is None or next((p for p in self.parameters if p.name == name), None) is None:
             raise KeyError(f"`{name}` is not a valid workflow parameter")
         return Parameter(name, value=f"{{{{workflow.parameters.{name}}}}}")
+
+    def to_dict(self) -> dict:
+        """Returns the dictionary representation of the workflow"""
+        return model_to_dict(self.build(), serialize=False)
+
+    def to_json(self) -> str:
+        """Returns the JSON representation of the workflow"""
+        return json.dumps(model_to_dict(self.build(), serialize=False))
```

Comment:
```suggestion
        return json.dumps(model_to_dict(self.build(), serialize=True))
```

Should this be true?

Reply:
Yes, added that and adjusted the test!